client/landing/jetpack-cloud/sections/scan/upsell.jsx

```diff
@@ -78,7 +78,7 @@ function renderUpsell( reason ) {
 export default function ScanUpsellPage( { reason } ) {
 	return (
 		<Main className="scan__main">
-			<DocumentHead title="Scanner" />
+			<DocumentHead title="Scan" />
 			<SidebarNavigation />
 			<PageViewTracker path="/scan/:site" title="Scanner Upsell" />
```

Comment:
Should we update this one as well?

Reply:
Yes, Scan and Backup are the names of the products

Reply:
Sorry. I meant the title of the `<PageViewTracker>` component.